models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -347,7 +353,9 @@ object implicits {
         case EAndBody(EAnd(p1, p2))     => p1.get.locallyFree | p2.get.locallyFree
         case EOrBody(EOr(p1, p2))       => p1.get.locallyFree | p2.get.locallyFree
         case EMethodBody(e)             => e.locallyFree
-        case ExprInstance.Empty         => BitSet()
+        case EMatchesBody(EMatches(target, pattern)) =>
+          target.locallyFree | pattern.locallyFree
```

Comment:
Same here.

Reply:
what would be a rholang test that will fail because of this?